src/consts.js

```diff
@@ -1,7 +1,4 @@
-const path = require('path')
-const process = require('process')
-
-module.exports.LOCAL_OUT_DIR = path.join(process.cwd(), '.netlify', 'edge-handlers')
```

Comment:
`process.cwd()` was unnecessary here since this file path is only used with core Node.js methods which handle any relative file path using `process.cwd()` as the default base.